tests/machine.test.js

```diff
@@ -1225,10 +1243,10 @@ describe(OlmMachine.name, () => {
             return secondMachine;
         }
 
-        test("Should request cross signing request if missing", async () => {
+        test("Should request cross signing-keys if missing", async () => {
```

Comment:
sorry, this was my bad:

```suggestion
        test("Should request cross-signing keys if missing", async () => {
```